mozilla_schema_generator/glean_ping.py

```diff
@@ -92,7 +92,7 @@ def get_dependencies(self):
                 self.dependencies_url_template.format(self.repo_name)
             )
         except HTTPError:
-            logging.info(f"For {self.repo_name}, using default Glean dependencies")
+            logging.info(f"No dependencies found for {self.repo_name}")
```

Comment:
A http error here means something went entirely wrong. Should we re-raise the exception here instead of silently failing?

Reply:
I'm not sure why we're catching this exception here, we don't do this in any other place `_get_json` is called. 
This method returns an empty dictionary for repositories that do not have dependencies defined. Maybe in the past probe-info-service returned an error in such cases?

I checked logs further back in time - I don't see this message logged in the last 365 days (except for `accounts-backend` and `moso-mastodon-backend`). Because the same message is logged in two places I cannot say where was it logged, but my testing suggests these were all from couple of lines below.

I think it makes sense to not catch this exception here and let it bubble up to the calling method. I'll push a commit with this change.